docs/book/src/quick-start.md

```diff
@@ -273,6 +273,8 @@ kind: Pod
 metadata:
   name: quick-start
   namespace: ${SERVICE_ACCOUNT_NAMESPACE}
+  labels:
+    azure.workload.identity/use: "true"
```

Comment:
This is just so that we start encouraging folks to use this label now right?  We have not made any other code changes right?

Reply:
Correct! We haven't made any code changes yet.